### DISCUSSION TITLE
Fixtures for a time range instead of exact date?

### DISCUSSION THREAD

**Discussion by adambernath** [cloudbet/docs#41] [Ideas]
What is showing on the highlights page of specific sports eg. https://www.cloudbet.com/en/esports/esport-valorant/highlights
looks like something that only could be replicated for affiliates with a ton of requests - as fixtures only come in an exact date packages. 
I'm not sure why its packed this way - or if theres any way to get multiple days in one request (no mentioning in the mini docs) - but I really need an upcoming events breakdown (eg. events next month from current date).

**Comment by kgravenreuth** [comment]
Hi @adambernath 

Thanks for this feedback. 

Currently, we provide this API based on dates, but we're discussing this internally and I'll update here about whether we decide to make a change to this API endpoint within a few days.

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Comment by adambernath** [comment]
Hi Klaus,
Thank you, it would definitely make my and probably a lot more peoples
lives easier if we could get the data with one request instead of 7 for a
week or 30 for a month.
Bests,
Adam

**Comment by kgravenreuth** [comment]
Hi @adambernath

Happy to let you know that we have added the `from` and `to` query parameters to our [`/v2/odds/fixtures` endpoint](https://www.cloudbet.com/api/?utm_source=github.com&utm_medium=social&utm_campaign=axis&urls.primaryName=Feed#/API-Version-2.0/GetFixtures) You should use these instead of the `date` query parameter if you want to query fixtures for a given sport by time range.

In addition, we've recently also added the [`/v2/odds/events` endpoint](https://www.cloudbet.com/api/?utm_source=github.com&utm_medium=social&utm_campaign=axis&urls.primaryName=Feed#/API-Version-2.0/GetEvents) which allows you to query live or upcoming event markets by time range too.

Let me know if you have any more questions.

Cheers,
Klaus
